### ISSUE TITLE
new gap ebuild: some documentation files are missing

### ISSUE THREAD

**Issue by kiwifb** [cschwan/sage-on-gentoo#411]
Gap output:
```
gap> ?help
Help: no matching entry found
```
Corresponding output of strace
```
read(0, "\33", 1)                       = 1
read(0, "[", 1)                         = 1
read(0, "A", 1)                         = 1
write(1, "?help", 5)                    = 5
write(1, "\10", 1)                      = 1
write(1, "\10", 1)                      = 1
write(1, "\10", 1)                      = 1
write(1, "\10", 1)                      = 1
write(1, "\10", 1)                      = 1
read(0, "\r", 1)                        = 1
write(1, "\n", 1)                       = 1
ioctl(0, SNDCTL_TMR_STOP or SNDRV_TIMER_IOCTL_GINFO or TCSETSW, {B38400 opost isig icanon echo ...}) = 0
rt_sigaction(SIGWINCH, {0x53237a, [WINCH], SA_RESTORER|SA_RESTART, 0x7fc1707ba310}, {0x7fc171159d46, [], SA_RESTORER|SA_RESTART, 0x7fc1707ba310}, 8) = 0
getrusage(RUSAGE_SELF, {ru_utime={1, 335000}, ru_stime={0, 66000}, ...}) = 0
access("/usr/lib64/gap/doc/tut/manual.six", F_OK) = -1 ENOENT (No such file or directory)
access("/usr/lib64/gap/doc/tut/manual.six.gz", F_OK) = -1 ENOENT (No such file or directory)
access("/usr/lib64/gap/doc/ref/manual.six", F_OK) = -1 ENOENT (No such file or directory)
access("/usr/lib64/gap/doc/ref/manual.six.gz", F_OK) = -1 ENOENT (No such file or directory)
access("/usr/lib64/gap/doc/changes/manual.six", F_OK) = -1 ENOENT (No such file or directory)
access("/usr/lib64/gap/doc/changes/manual.six.gz", F_OK) = -1 ENOENT (No such file or directory)
write(1, "Help: no matching entry found\n", 30) = 30
```
and `manual.six` is indeed absent. It is not on github either so it is probably generated in the release process. I may have to go back to producing a lite tarball.

**Comment by kiwifb** [comment]
Worse than that. We only have `.xml` files no `html` or anything.

**Comment by kiwifb** [comment]
Also missing the whole `tst` folder.

**Comment by kiwifb** [comment]
Fixed by 4fbc075ffeea8ebec058c1a7f4a7aa57bf7c13cd
